api-description/forms/pk/post.json

```diff
@@ -1,6 +1,6 @@
 {
     "description": "Endpoint to answer a form",
-    "url": "http://localhost:8000/forms/1",
+    "url": "http://localhost:8000/forms/1/respond/",
```

Comment:
`/create` suggests you're making a new form, which you aren't here - you're submitting a user's response to it.